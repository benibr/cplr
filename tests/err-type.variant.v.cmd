../bin/cplr '-n' 'puts(1)' -v
