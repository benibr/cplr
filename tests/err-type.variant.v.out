Generation phase
Generating code
Generating section defsysinclude
Generating section main
Generated: 1029 bytes code, 0 bytes dump
Compilation phase
Compiler condition:
main_0:1: warning: assignment makes pointer from integer without a cast
Compilation succeeded
